src/styles/1-helpers/_variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
+$white: #fff;
+$blue: #00acdc;
+$black: #000;
+$grey: #eee;
```

Comment:
It is better to name the variables not according to the content, but according to the meaning, because the color may change during the development process and it will have to be corrected in the entire file.

Reply:
done